source/index.html.md

```diff
@@ -637,6 +638,12 @@ All events posted to the designated URL fit the same structure.
 <aside class="notice">The sandbox environment allow both HTTP and HTTPS webhook URLs. The live environment however will only POST to HTTPS URLs.
 </aside>
 
+### Our delivery promises
+1. A webhook event delivery is only considered failed if we don't receive any http response code (2xx, 4xx, 5xx, etc...)
```

Comment:
Consider active over passive voice, e.g.
We only consider a webhook event delivery as failed if we don't receive any http response code (2xx, 4xx, 5xx, etc.)